tools/circexplorer/circexplorer.xml

```diff
@@ -0,0 +1,102 @@
+<tool id="circexplorer" name="CIRCexplorer" version="0.1.0">
+    <requirements>
+        <requirement type="package" version="2.1.0">tophat</requirement>
+        <requirement type="package" version="0.6.2">docopt</requirement>
+        <requirement type="package" version="0.8.4">pysam</requirement>
+    </requirements>
+    <command><![CDATA[
+        #if str($input_selection.input_type) == "star"
+            python $__tool_directory__/circ/star_parse.py "$input_selection.junc" ./fusion_junctions.txt &&
+        #end if
+
+        python $__tool_directory__/circ/CIRCexplorer.py
+
+        #if str($input_selection.input_type) == "tophat"
+            -f "$input_selection.fusion"
+        #elif str($input_selection.input_type) == "star"
+            -j ./fusion_junctions.txt
+        #end if
+
+        -g "$genome"
+        -r "$ref"
+        $no_fix
+]]>
+    </command>
+    <inputs>
+        <conditional name="input_selection">
+            <param name="input_type" type="select" label="Select input type" help="">
+                <option value="tophat">TopHat &amp; TopHat-Fusion (--fusion, --genome and --ref)</option>
+                <option value="star">STAR (--junc, --genome, and --ref)</option>
+            </param>
+            <when value="tophat">
+                <param name="fusion" type="data" format="bam" label="TopHat-Fusion fusion BAM file (used in TopHat-Fusion mapping)" help="(--fusion)" />
+            </when>
+            <when value="star">
+                <param name="junc" type="data" format="bam" label="STAR Chimeric junction file (used in STAR mapping)" help="(--junc)" />
+            </when>
+        </conditional>
+
+        <param name="genome" type="data" format="fasta" label="Genome FASTA file" help="(--genome)" />
```

Comment:
Tipp: If you use `--argument="--genome"` you can omit the `name` and the `help` it will be automatically generated.